app/src/test/java/com/x8bit/bitwarden/data/autofill/processor/AutofillCipherProviderTest.kt

```diff
@@ -118,7 +120,10 @@ class AutofillCipherProviderTest {
                 autofillCipherProvider.isVaultLocked()
             }
 
-            testScheduler.advanceUntilIdle()
+            assertFalse(result.isCompleted)
+            testScheduler.advanceTimeBy(delayTimeMillis = 1_000L)
```

Comment:
I don't think we need this here. You should call `runCurrent` before the `isCompleted` check because I would think this is already completed.